circle.yml

```diff
@@ -17,7 +14,7 @@ executors:
       - image: ethereum/cpp-build-env:14-gcc-10
   linux-clang-latest:
     docker:
-      - image: ethereum/cpp-build-env:14-clang-10
+      - image: ethereum/cpp-build-env:15-clang-10
```

Comment:
What is the update in 15? Why not update the others for consistency too?

Reply:
I only updated this one to include `clang-format`.